src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,100 @@
+@import '../utils/mixins';
+@import '../utils/variables';
+
+.card {
+  @include flex-center;
+
+  width: 200px;
+  padding: 0 16px 16px;
+  background-color: $white-color;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+
+  transition:
+    background-color 300ms ease,
+    color 300ms ease,
+    border 300ms ease;
+
+  &__img {
+    margin-top: 32px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    @include product-title-font;
+
+    margin-top: 40px;
+  }
+
+  &:hover {
```

Comment:
The `transform: scale(1.2);` on hover might make the content of the card difficult to read for users. Consider a more subtle effect, such as a slight change in shadow or border, to indicate hover state without affecting readability.